bundles/tools.vitruv.applications.umljava/META-INF/MANIFEST.MF

```diff
@@ -5,24 +5,15 @@ Bundle-SymbolicName: tools.vitruv.applications.umljava;singleton:=true
 Automatic-Module-Name: tools.vitruv.applications.umljava
 Bundle-Version: 0.2.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-11
-Require-Bundle: com.google.guava,
- org.apache.log4j,
- org.eclipse.emf.ecore,
- org.eclipse.core.resources,
- org.eclipse.xtext.xbase.lib,
+Require-Bundle: org.apache.log4j,
  org.eclipse.xtend.lib,
- org.eclipse.xtend.lib.macro,
  edu.kit.ipd.sdq.activextendannotations,
  tools.vitruv.domains.uml,
  tools.vitruv.domains.java,
  tools.vitruv.framework.applications,
  tools.vitruv.framework.change,
- tools.vitruv.framework.change.processing,
- tools.vitruv.framework.correspondence,
- tools.vitruv.framework.userinteraction,
- tools.vitruv.framework.util,
- tools.vitruv.extensions.dslsruntime.reactions,
```

Comment:
These dependencies are required by the Reactions. Removing them will throw warnings.

Reply:
This also applied to `com.google.guava`.